megatron/checkpointing.py

```diff
@@ -152,7 +152,22 @@ def save_ds_checkpoint(iteration, model, neox_args):
         logits = do_forward_pass(neox_args=neox_args, model=model)
         sd['checkpoint_validation_logits'] = logits
     
-    model.save_checkpoint(neox_args.save, client_state=sd)
+    # Derive tag 
+    # The tag is usually automatically derived in the same way in deepspeed
+    # As the save_checkpoint function does not return such derived tag, we 
+    # do it manually in order to use the subdirectory to save config files
+    tag = f"global_step{iteration}"
```

Comment:
I would say these comments are a bit more verbose than necessary - I think what's happening here is straightforward enough that we can just say something like `# Derive tag` or `# checkpoint folder name` and save a few LOC

Reply:
No opinion really. Can change :-)

Reply:
done